### ISSUE TITLE
label test

### ISSUE THREAD

**Issue by iimpulse** [obophenotype/human-phenotype-ontology#11011]
**Preferred term label:** fake label test

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
